setup.py

```diff
@@ -7,6 +7,7 @@
     name="diptest._diptest",
     sources=["diptest/_dip.c", "diptest/_diptest.pyx"],
     extra_compile_args=['-O3', '-std=c99'],
+    ininclude_dirs=[np.get_include()],
```

Comment:
This is a typo

Reply:
sorry... It is too long away that I cannot what I did...let me see...